### PR TITLE
[Feature] Add quantum dropout 

### PR DESCRIPTION
**Pull request by chMoussa**
Solves #530

### CODE REVIEW

File: qadence/backends/pyqtorch/config.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from logging import getLogger
 from typing import Callable
 
-from pyqtorch.utils import SolverType
+from pyqtorch.utils import DropoutMode, SolverType
```
Comment (jpmoutinho) [inline qadence/backends/pyqtorch/config.py:7]: You need to expose the `DropoutMode` in Qadence by adding it to the `types` as well.

**Comment by jpmoutinho** [comment]
I added a line here to keep track of documenting this: https://github.com/pasqal-io/qadence/issues/535

**Review by jpmoutinho — APPROVED** [review]
Thanks, lgtm :)
